04-vue-cli/02-TheToaster/components/TheToast.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="['toast', data.className]">
+    <UiIcon class="toast__icon" :icon="data.icon" />
+    <span>{{ text }}</span>
+  </div>
+</template>
+
+<script lang="ts">
+import UiIcon from './UiIcon.vue';
+import { defineComponent } from 'vue';
+
+export enum EToastKind {
+  SUCCESS = 'SUCCESS',
+  ERROR = 'ERROR',
+}
+
+type TToastData = {
+  icon: string;
+  className: string;
+};
+
+export default defineComponent({
+  name: 'TheToast',
+
```

Comment:
Components are usually named with the `The` prefix when they are supposed to be used only with exactly one instance.